dist.ini

```diff
@@ -0,0 +1,68 @@
+name    = Pod-Usage
+author  = Marek Rouchal <marekr@cpan.org>
+license = Perl_5
+copyright_holder = Marek Rouchal <marekr@cpan.org>
+copyright_year   = 2013
+
+[ReadmeAnyFromPod / Markdown_Readme]
+type = gfm
+source_filename = lib/Pod/Usage.pm
+filename = README.md
+location = root
+
+[Regenerate::AfterReleasers]
+plugin = Markdown_Readme
+
+[@Starter::Git]
+revision = 4
+managed_versions = 1
+installer = MakeMaker::Awesome
+-remove = MakeMaker::Awesome
+RewriteVersion.global = 1
+NextRelease.format = %-9v %{yyyy-MM-dd}d
+regenerate = Makefile.PL
+regenerate = META.json
+regenerate = README.md
+regenerate = LICENSE
+regenerate = t/00-report-prereqs.t
```

Comment:
This test requires its associated data file, `t/00-report-prereqs.dd`

Reply:
thanks, added by 700408a18276f21661647aaecb2ba6a559a8429e

Reply:
Does this test even need to be in git? I don't see the point. The test's advantage is having more data show up in cpantesters reports.

Reply:
I steal it from https://github.com/Dual-Life/mime-base64/commit/4ae1f3db3c380aa6b03f2b2dbdf90fcb0d55dc1e 
but as @Grinnz noted this was incomplete as the `.dd` file was missing.
I can see it going both way. The main advantage of having it versioned is to help user just forking the git repo and trying to run test. Otherwise the value is low.